pages/desktop/base.py

```diff
@@ -52,6 +59,13 @@ def format_page_title(self, *title_segments):
             page_title = title + separator + page_title
         return page_title
 
+    class FooterRegion(Page):
+
+        _switch_to_mobile_site = (By.CSS_SELECTOR, 'div.grid_2:nth-child(4) > ul:nth-child(1) > li:nth-child(4) > a:nth-child(1)')
```

Comment:
This locator is pretty nasty. I believe that this will work:

`(By.CSS_SELECTOR, 'footer a[href*="mobile=1"]')`

That says to find an `a` tag that is in the `footer` element and which has an `href` that includes the text `mobile=1`.